Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -207,6 +207,29 @@ extension StripeCardReaderService: CardReaderService {
         }
     }
 
+    public func waitForInsertedCardToBeRemoved() -> Future<Void, Error> {
+        return Future() { [weak self] promise in
+            guard let self = self else {
+                return
+            }
+
+            // If there is no chip card inserted, it is ok to immediately return. The payment method may have been swipe or tap.
+            if !self.isChipCardInserted {
```

Comment:
nit: Maybe this could be a guard:

```
guard self.isChipCardInserted else {
     return promise(.success(()))
}
```

Reply:
Nice catch! updated as part of https://github.com/woocommerce/woocommerce-ios/pull/6665/commits/f0b4a6f5e5c1845d840be7444a21dd483f433442